baremetal/install.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+# Trusted Compute Installation Script
+# This script installs trusted-compute components for K3s
+# Must be run as sudo
+
+set -e  # Exit on any error
+
+# Color codes for output
+RED='\033[0;31m'
```

Comment:
check if \033 is better way or \e  i.e 
RED="\e[31m"
GREEN="\e[32m"
ENDCOLOR="\e[0m"